packages/animations/CHANGELOG.md

```diff
@@ -1,18 +1,18 @@
-## [1.0.0+3] - February 18, 20200
+## [1.0.0+3] - February 18, 2020
```

Comment:
If you don't want to wait for another push to pub.dev, you'll need to bump the patch version to +4 so you can republish the fixed CHANGELOG there.